### PR TITLE
Win2012 test fix

### PR DESCRIPTION
**Pull request by dchappa**
# Description of the issue
Resolves issue with windows 2012 acceptance and restart testing not passing

# Description of changes
The `-NoExit` parameter causes the command to not exit after finishing, ultimately timing out. I've removed the calls from `NoExit` from the functions that are called from these tests

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
Integration testing

### CODE REVIEW

File: terraform/ec2/win/main.tf [hunk 1]
```diff
@@ -153,6 +153,7 @@ resource "null_resource" "integration_test_run" {
     user     = "Administrator"
     password = rsadecrypt(aws_instance.cwagent.password_data, local.private_key_content)
     host     = aws_instance.cwagent.public_dns
+    timeout = "5m"
```
Comment (movence) [inline terraform/ec2/win/main.tf:156]: nit: we can remove this since 5m is the default?

**Review by movence — APPROVED** [review]

File: terraform/ec2/win/main.tf (hunk 1 above)
Comment (dchappa) [reply terraform/ec2/win/main.tf]: Good point, was tinkering with the timeouts before. Recent commit addresses that.

File: terraform/ec2/win/main.tf [hunk 2]
```diff
@@ -168,6 +168,34 @@ resource "null_resource" "integration_test_run" {
...
+
+  provisioner "file" {
+    source      = module.validator.agent_config
+    destination = module.validator.instance_agent_config
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "set AWS_REGION=${var.region}",
+      var.use_ssm ? "powershell \"& 'C:\\Program Files\\Amazon\\AmazonCloudWatchAgent\\amazon-cloudwatch-agent-ctl.ps1' -a fetch-config -m ec2 -s -c ssm:${local.ssm_parameter_name}\"" : "powershell \"& 'C:\\Program Files\\Amazon\\AmazonCloudWatchAgent\\amazon-cloudwatch-agent-ctl.ps1' -a fetch-config -m ec2 -s -c file:${module.validator.instance_agent_config}\"",
```
Comment (sky333999) [inline terraform/ec2/win/main.tf:193]: I see we already start the agent here.. so why is this needed again?

Comment (dchappa) [reply terraform/ec2/win/main.tf]: that may have been a holdover from me trying to root cause the issue. Running tests again after removing it

**Review by movence — APPROVED** [review]

**Review by sky333999 — APPROVED** [review]
